packages/simulator/tests/server.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { WalletAPIClient } from "@ledgerhq/wallet-api-client";
+import { BitcoinTransaction, schemaFamilies } from "@ledgerhq/wallet-api-core";
+import BigNumber from "bignumber.js";
+import { getSimulatorTransport, profiles } from "../src";
+
+function createBitcoinTx(): BitcoinTransaction {
+  return {
+    family: schemaFamilies.enum.bitcoin,
+    amount: new BigNumber(0),
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    recipient: profiles.STANDARD.accounts[0]!.address,
```

Comment:
We can probably use `as const` in the profiles to avoid that kind of issue with typescript

Reply:
I casted `as Account` instead.

<img width="586" alt="Screenshot 2023-03-07 at 15 26 34" src="https://user-images.githubusercontent.com/8398372/223451199-e2c93cdd-9ee5-461b-b9bb-d475f21b628a.png">